tests/ext/flask_sqlalchemy/test_query.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import absolute_import
+import json
+import jsonpickle
+import pytest
+from aws_xray_sdk.core import xray_recorder
+from aws_xray_sdk.core.context import Context
+from aws_xray_sdk.ext.flask_sqlalchemy.query import XRayFlaskSqlAlchemy
+from flask import Flask
+
+
+app = Flask(__name__)
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:///:memory:"
+db = XRayFlaskSqlAlchemy(app)
+
+
+class User(db.Model):
+    __tablename__ = "users"
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(255), nullable=False, unique=True)
+    fullname = db.Column(db.String(255), nullable=False)
+    password = db.Column(db.String(255), nullable=False)
+
+
+def _search_entity(entity, name):
```

Comment:
This along with find_sub could also be a helper class shared between your two query_test.py.  And since these two functions are very general they could be under tests/ root so any other unit test can utilize your contribution.